cypress.config.ts

```diff
@@ -13,7 +21,192 @@ export default defineConfig({
     POSTGRES_URL: 'postgresql://postgres:postgres@localhost:5432/registry',
   },
   e2e: {
-    setupNodeEvents(on) {
+    setupNodeEvents(on, config) {
```

Comment:
Could we potentially re-use the testkit we have within our integration tests here?

Reply:
yeah, we should